networkx/readwrite/json_graph/node_link.py

```diff
@@ -1,18 +1,21 @@
-#    Copyright (C) 2011-2013 by
+#    Copyright (C) 2011-2016 by
+# 
 #    Aric Hagberg <hagberg@lanl.gov>
 #    Dan Schult <dschult@colgate.edu>
 #    Pieter Swart <swart@lanl.gov>
+#    Michael E. Rose <Michael.Ernst.Rose@gmail.com>
+#    
 #    All rights reserved.
 #    BSD license.
 from itertools import chain, count
 import json
 import networkx as nx
 from networkx.utils import make_str
-__author__ = """Aric Hagberg <hagberg@lanl.gov>"""
 __all__ = ['node_link_data', 'node_link_graph']
 
 
-_attrs = dict(id='id', source='source', target='target', key='key')
+_attrs = dict(source='source', target='target', name='name',
+              key='key', link='links')
 
 
 def node_link_data(G, attrs=_attrs):
```

Comment:
Using a mutable default value for a keyword argument will cause problems. Use `attrs=None` instead, and have a special check at the beginning of the function for `if attrs is None: ...`.